Samples.xcodeproj/project.pbxproj

```diff
@@ -2960,7 +2960,6 @@
 				};
 			};
 			buildConfigurationList = 00E5400A27F3CCA100CF66D5 /* Build configuration list for PBXProject "Samples" */;
-			compatibilityVersion = "Xcode 15.0";
```

Comment:
Normally Xcode would automatically update this to "Xcode 16.0". Did that not happen?

Reply:
I didn't receive the "Update to suggested project setting" warning on the first launch. So instead, I triggered one manually with Eskimo's [answer](https://forums.developer.apple.com/forums/thread/743932).

Reply:
This isn't related to that. Xcode should change this when the project format is changed:

![image](https://github.com/user-attachments/assets/0ae9e557-3a85-4c10-ba57-83b4d9657806)

Reply:
I didn't see that happen. Just now I tried to select 15.0, and there isn't a change in the project file after I select back to 16.0.

Reply:
Fascinating. Something must have changed in Xcode 16.